contact/contact-script.js

```diff
@@ -0,0 +1,10 @@
+$(function() {
+  $(".contact-tpl__button-1st").click(function() {
+    var $button-2nd_wrapper = $(this).find(".answer");
+    if ($button-2nd_wrapper.hasClass("open")) {
+      $button-2nd_wrapper.removeClass("open");
+    } else {
+      $button-2nd_wrapper.addClass("open");

```

Comment:
```suggestion
    var $button2nd_wrapper = $(this).find(".answer");
    if ($button2nd_wrapper.hasClass("open")) {
      $button2nd_wrapper.removeClass("open");
    } else {
      $button2nd_wrapper.addClass("open");
```

演算のマイナスと見られるため、JSの変数名に、ハイフン `-` は使えないです。
同様に、 プラス `+` などの演算記号は使用できません。

>JavaScript の識別子は必ずアルファベットかアンダースコア (_) かドル記号 ($) から始まらなくてはなりません。続く文字は数字 (0-9) も使えます。JavaScript は大文字・小文字を区別するため、使えるアルファベットは "A" から "Z"（大文字）と "a" から "z"（小文字）です。